roles/activemq/tasks/main.yml

```
@@ -13,14 +13,9 @@
     mode: "0755"
   register: download_path
 
-- name: Download ActiveMQ archive # noqa ignore-errors
-  delegate_to: localhost
-  ansible.builtin.get_url:
-    url: "{{ dependencies_url.activemq }}"
-    checksum: sha512:{{ dependencies_url.activemq_sha512_checksum_url }}
-    dest: "{{ download_path.path }}/apache-activemq-{{ activemq_version }}-bin.tar.gz"
-    mode: "0644"
-  ignore_errors: true
+- name: Download ActiveMQ {{ activemq_version }} archive from {{ activemq_url }}
```

```suggestion
- name: Download ActiveMQ archive 
```
Not needed - but I think there is a possible value